common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -1038,6 +1029,7 @@ def test_video_with_no_subs_index_dictionary(self):
             "content_type": "Video"
         })
 
+    @httpretty.activate
```

Comment:
On the master PR, this test and the other below are marked as skipped.  Why is it different for Ironwood?

Reply:
@nedbat They were temporarily skipped because that was the fastest solution to get other work unblocked, but a later PR unskipped and fixed them: https://github.com/edx/edx-platform/pull/23680